test/suites/edgexfoundry/proxy_test.go

```diff
@@ -0,0 +1,129 @@
+package test
+
+import (
+	"crypto/tls"
+	"edgex-snap-testing/test/utils"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var kongAdminJwtFile = "/var/snap/edgexfoundry/current/secrets/security-proxy-setup/kong-admin-jwt"
+
+func TestAddProxyUser(t *testing.T) {
+	var (
+		tmpDir     = "keys"
+		publicKey  = tmpDir + "/public.pem"
+		privateKey = tmpDir + "/private.pem"
+	)
+
+	// start clean
+	err := os.RemoveAll(tmpDir)
+	require.NoError(t, err)
+
+	t.Cleanup(func() {
+		err = os.RemoveAll(tmpDir)
+		require.NoError(t, err)
+	})
+
+	// Get Kong admin JWT token
+	out, err := os.ReadFile(kongAdminJwtFile)
+	require.NoError(t, err)
```

Comment:
If this is the only step requiring sudo, you can copy the token to local directory and change its permission to allow reads with Go's ReadFile.

```
sudo install -m 604 /var/snap/edgexfoundry/current/secrets/security-proxy-setup/kong-admin-jwt .
```